src/content/glossary/index.md

```diff
@@ -87,6 +91,26 @@ A collection of required information (a block header) about the comprised [trans
   Blocks
 </DocLink>
 
+### block(chain) explorer
+
+A website that allows easy searching and extraction of data from the blockchain.
```

Comment:
This seems vague.

Reply:
Changed to:
"A website that allows a user to search for information from, and about, a blockchain. This includes retrieving individual transactions, activity associated with specific addresses and information about the network."